### ISSUE TITLE
DensePose DeepLabHead setting is wired.

### ISSUE THREAD

**Issue by cool-xuan** [facebookresearch/detectron2#1995]
In [DensePoseDeepLabHead](https://github.com/facebookresearch/detectron2/blob/729290df8a1b9c0307fb667dd776bed3c4f8bdc5/projects/DensePose/densepose/densepose_head.py#L98), you set atrous_rates in [ASSP](https://github.com/facebookresearch/detectron2/blob/729290df8a1b9c0307fb667dd776bed3c4f8bdc5/projects/DensePose/densepose/densepose_head.py#L185) [6, 12, 56], while in [DeepLabv3](https://arxiv.org/pdf/1706.05587.pdf) atrous_rates is [6, 12, 18] and in your copied link [torchvision](https://github.com/pytorch/vision/blob/master/torchvision/models/segmentation/deeplabv3.py), the atrous_rates is [12, 24, 36] which is double to [6, 12, 24]. Are your settings designed for DensePose task or some other reasons?

Meanwhile, you replace batch normalization by group normalization, while deeplabv3 paper and torchvision code both use batch normalization. Is there some reason to use group normalization? Maybe you make some experiments which prove GN is better?

**Comment by vkhalidov** [comment]
@cool-xuan there hasn't been any thorough study on the optimal "a trous" layout or normalization type
